rippleapi_quickstart.html

```diff
@@ -0,0 +1,520 @@
+<h4 id="2-install-dependencies-using-npm">2. Install dependencies using NPM</h4>
+<p>You need to have <a href="#install-nodejs-and-npm">NPM (Node.js Package Manager) installed</a> first.</p>
+<p>Then, from within the <code>ripple-lib</code> directory, you can use NPM to install all the necessary dependencies:</p>
+<pre><code>npm install
+</code></pre>
+<p>(We recommend <em>not</em> using <code>npm -g</code> to install dependencies globally.)</p>
+<p>This can take a while, and may include some warnings. The following warnings are benign and do not indicate a problem:</p>
+<pre><code>npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
+npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.6
+</code></pre>
+<h4 id="3-use-gulp-to-build-a-single-javascript-output">3. Use Gulp to build a single JavaScript output</h4>
+<p>RippleAPI comes with code to use the <a href="http://gulpjs.com/">gulp</a> package to compile all its source code into browser-compatible JavaScript files. Gulp is automatically installed as one of the dependencies, so all you have to do is run it:</p>
+<pre><code>./node_modules/.bin/gulp
+</code></pre>
+<p>This may take a little while. It outputs several things, and creates a new <code>build/</code> folder, which contains the files you want.</p>
+<p>The file <code>build/ripple-&lt;VERSION NUMBER&gt;.js</code> is a straight export of RippleAPI (whatever version you built) ready to be used in browsers. The file ending in <code>-min.js</code> is the same thing, but with the content <a href="https://en.wikipedia.org/wiki/Minification_%28programming%29">minified</a> for faster loading.</p>
+<h2 id="example-browser-usage">Example Browser Usage</h2>
+<p>The following HTML file demonstrates basic usage of the browser version of RippleAPI to connect to a public <code>rippled</code> server and report information about that server. Instead of using Node.js's "require" syntax, the browser version creates a global variable named <code>ripple</code>, which contains the <code>RippleAPI</code> class.</p>
+<p>To use this example, you must first <a href="#build-instructions">build RippleAPI</a> and then copy one of the resulting output files to the same folder as this HTML file. (You can use either the minified or full-size version.) Modify the first <code>&lt;script&gt;</code> tag in this example to use the correct file name for the version of RippleAPI you built.</p>
+<pre><code>&lt;!DOCTYPE html&gt;
+&lt;html&gt;
+&lt;head&gt;
+  &lt;script src="ripple-0.16.5.js"&gt;&lt;/script&gt;
+  &lt;script&gt;
+    console.log(ripple);
+    var api = new ripple.RippleAPI({server:'wss://s1.ripple.com/'});
+    api.connect().then(()=&gt;{
+        return api.getServerInfo();
+    }).then(server_info=&gt;{
+    document.body.innerHTML += `&lt;p&gt;Connected to rippled server!&lt;/p&gt;
```

Comment:
I think you would have to get rid of the template string, which is an ES6 feature